### ISSUE TITLE
DUCE conencted to FERY

### ISSUE THREAD

**Issue by rgalland** [msinger/dmg-schematics#43]
Hi @msinger, can you understand why FERY is not connected to FF18?

**Comment by msinger** [comment]
Yes, I can. In Furrtek's schematics there is a connection, because it's hard to follow the wires here on the die shot. He just didn't see it correctly. Took me a long time too, to find this. I made two images in issue furrtek/DMG-CPU-Inside#62 already, that illustrate the problem.

**Comment by msinger** [comment]
So, this means you can also remove the outgoing hierarchical label DUCE on the APU decode sheet, I think.
